crates/sui-types/src/collection_types.rs

```diff
@@ -12,6 +12,18 @@ pub struct VecMap<K, V> {
     pub contents: Vec<Entry<K, V>>,
 }
 
+impl<K: PartialEq, V> VecMap<K, V> {
+    pub fn get(&self, key: &K) -> Option<&V> {
+        self.contents.iter().find_map(|entry| {
+            if &entry.key == key {
+                Some(&entry.value)
+            } else {
+                None
+            }
```

Comment:
```suggestion
            (&entry.key == key).then_some(&entry.value)
```

Reply:
You can't get bored with Rust. Whenever I think I've exhausted all the "builder-style" calls, there are more! 😆